### PR TITLE
Move Room/Client/RoomPage to separate files

### PR DESCRIPTION
**Pull request by chuckhays**
There are no functionality changes here, only moving Client, Room and RoomPage into separate files to separate things a bit better and reduce the size of apprtc.py.

The change is split into 2 commits so the moves are clearer if the combined view is hard to follow.

@tkchin @jiayliu

### CODE REVIEW

**Comment by chuckhays** [comment]
Updated. PTAL.

File: samples/web/content/apprtc/room.py [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+    wss_url = 'ws://' + ws_host_port_pair + '/ws'
+    wss_post_url = 'http://' + ws_host_port_pair
+  else:
+    wss_url = 'wss://' + ws_host_port_pair + '/ws'
+    wss_post_url = 'https://' + ws_host_port_pair
+  return (wss_url, wss_post_url)
+
+# Returns appropriate room parameters based on query parameters in the request.
+# TODO(tkchin): move query parameter parsing to JS code.
+def get_room_parameters(request, room_id, client_id, is_initiator):
```
Comment (jiayliu) [inline samples/web/content/apprtc/room.py]: I'd keep this one and the other parameter related methods in apprtc.py, since they really have nothing to do with Room. Or put them in a separate file.

File: samples/web/content/apprtc/room_test.py [hunk 2]
```diff
@@ -20,21 +21,21 @@ def tearDown(self):
     self.testbed.deactivate()
 
   def testAllowedClientList(self):
-    room = apprtc.Room(apprtc.Room.TYPE_DIRECT)
+    clientRoom = room.Room(room.Room.TYPE_DIRECT)
```
Comment (jiayliu) [inline samples/web/content/apprtc/room_test.py]: nit: rename clientRoom

File: samples/web/content/apprtc/room.py (hunk 1 above)
Comment (chuckhays) [inline samples/web/content/apprtc/room.py]: OK, I'll have to move them to a separate file or we would have a circular import.

**Comment by chuckhays** [comment]
Update with your suggestions. PTAL.
